src/main/java/com/corndel/cashregister/Level2.java

```diff
@@ -14,15 +14,13 @@ public class Level2 {
    * N.b. $1 is a note, not a coin
    */
   public static int countCoins(List<Item> drawer) {
-    // TODO
-    return 0;
+    return drawer.stream().filter(x -> drawer.indexOf(x) > 4).map(x->x.quantity).reduce(0, Integer::sum);
```

Comment:
Oh wow nice use of streams and reduce!

I would make one tweak and that is not to use the index as your predicate for your filter. 

You could get a drawer in a different order.

```java
return drawer.stream().filter(x -> x.getValue() < 100).map(x->x.quantity).reduce(0, Integer::sum);
```